boot/zephyr/main.c

```diff
@@ -175,6 +183,40 @@ static void do_boot(struct boot_rsp *rsp)
 }
 #endif
 
+#if defined(CONFIG_LOG) && !defined(CONFIG_LOG_IMMEDIATE) &&\
+    !defined(CONFIG_LOG_PROCESS_THREAD)
+#include <logging/log_ctrl.h>
```

Comment:
Why is it necessary to do this twice instead of just once, outside of `CONFIG_SOC_FAMILY_NRF`?